src/status/pollingClient.ts

```diff
@@ -54,19 +55,28 @@ export class PollingClient extends AsyncOptionalCreatable<PollingClient.Options>
    * Returns a promise to call the specified polling function using the interval and timeout specified
    * in the polling options.
    */
-  public async subscribe(): Promise<AnyJson | undefined> {
+  public async subscribe<T>(): Promise<T> {
```

Comment:
```suggestion
  public async subscribe<T = AnyJson>(): Promise<T | undefined> {
```
Then people don't have to supply a type (it's always been AnyJson) unless they want to.

I suggest keeping the `undefined` to avoid breaking the contract, even if we're not using it.
And leave a comment explaining why it's there, with a TODO: remove for v3